app/src/main/java/com/github/ybecker/epforuml/database/MockDatabase.kt

```diff
@@ -75,10 +77,23 @@ class MockDatabase : Database() {
         val answer5 = Answer("answer5", "question1", "user1", "Nan mais je suis pas d'accord non plus")
         addAnswer(answer5.userId, answer5.questionId, answer5.answerText)
 
+        val chat1 = Chat("chat0",LocalDateTime.now().toString(), user1.userId, user1.userId, "Hey me!")
+        addChat(chat1.senderId, chat1.receiverId, chat1.text)
         this.addSubscription(user1.userId, course1.courseId)
         this.addSubscription(user1.userId, course2.courseId)
 
     }
+    override fun getChat(userId1: String, userId2:String): CompletableFuture<List<Chat>> {
```

Comment:
This function has the same name as the private getChat method, which could cause confusion